frontend/src/routes/Video.tsx

```diff
@@ -515,6 +520,24 @@ const ShareButton: React.FC<{ event: SyncedEvent }> = ({ event }) => {
         ))}
     </div>;
 
+    const QRComponent: React.FC<{ target: string; label: State }> = ({ target, label }) => <>
+        <Button
+            onClick={() => currentRef(qrModalRef).open()}
+            css={{ width: "max-content" }}
+        >
+            <HiOutlineQrCode />
+            {t("video.share.show-qr-code")}
+        </Button>
+        <Modal ref={qrModalRef}
+            title={t("video.share.title", { title: label })}
+            css={{ minWidth: "max-content" }}
+        >
+            <div css={{ display: "flex", justifyContent: "center" }}>
+                <QRCodeCanvas value={target} size={250}/>
```

Comment:
Weird. I was somehow under the impression that the canvas was needed to make the thing copyable... but that's apparently not the case🧐. Thank you for checking.

Reply:
Lol I didn't check that :D That would have been a good argument and I would have also assumed that you can't copy the image then. 

Right now, you switched to just `<QRCode>`, which as per [README of that library](https://www.npmjs.com/package/qrcode.react) is deprecated. So you should use either `QRCodeCanvas` or `QRCodeSVG`. Assuming that SVG can indeed not be copied easily via the browser, I'm totally fine with `QRCodeCanvas`, i.e. as before. Just wanted to ask.

Reply:
I meant thanks for checking what's recommended in the library ;) It's not like I didn't take a look when choosing it but yesterday I kinda took the shortcut.

> And it also uses a canvas

Well that explains why I didn't notice any change when using `<QRCode>` instead of `<QRCodeCanvas>`...
But yeah as it turns out, my first impression and reasoning was correct for once, so I think we should stick with `<QRCodeCanvas>`.